s/api/services/admin-actions-service.ts

```diff
@@ -3,19 +3,26 @@ import * as dbmage from "dbmage"
 import * as renraku from "renraku"
 
 import {Auth} from "../types/auth.js"
+import {BanUser, UnBanUser} from "../types/concepts.js"
+import {enforceValidation} from "./utils/enforce-validation.js"
+import {validateAdminBanUser} from "./validators/validateAdminBanUser"
```

Comment:
all 'direct' es module imports must include `.js` extensions -- these will throw errors

please add .js on all imports of other files